_episodes/01-metadata.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Reporting to the Database
+teaching: 5
+exercises: 0
+questions:
+    - "What is the Ocean Tracking Network?"
+    - "How does your local telemetry network interact with OTN?"
+    - "What methods of data analysis will be covered?"
+---
+## Reporting Data to a OTN Node
+
+As researchers who are part of the MigraMar Network, you are encouraged to register your projects and report your data and metadata in a timely manner to your Data Manager. This will benefit all researchers in the region through the database's detection-matching system.
+
+This presentation **LINK TO PPT HERE** will cover some of the following topics.
+
+You are encouraged to read the [OTN FAQs Page](https://members.oceantrack.org/faq) for more information.
+
+### How to register with the MigraMar Database?
+
+If you wish to register your project with MigraMar you can email OTNDC@DAL.CA for assistance. We require 3 metadata types: 1) project metadata, 2) instrument deployment metadata and 3) tagging metadata. See the templates [here](https://members.oceantrack.org/data/data-collection). These will soon be translated to Spanish for your use!
+
+### What is the benefit of registering with the MigraMar Database?
+
+MigraMar and affiliated networks provide automated cross-referencing of your detection data with other tags in the system to help resolve "mystery detections" and provide detection data to taggers in other regions. MigraMar's Data Manager will also extensively quality control your submitted metadata for errors to ensure the most accurate records possible are stored in the database. MigraMar's database and Data Portal website are excellent places to archive your datasets for future use and sharing with collaborators. We offer pathways to publish your datasets with [OBIS](https://obis.org/), and via open data portals like ERDDAP, GeoServer etc. The data-product format returned by MigraMar is directly ingestible by analysis packages such as [glatos](https://github.com/ocean-tracking-network/glatos) and [resonATe](https://gitlab.oceantrack.org/otndc/resonate) for ease of analysis. OTN offers support for the use of these packages and tools.
```

Comment:
Should 'mystery detections' be in quotes?

Reply:
its OTN-wording for unknown tags. other groups call them "orphan tags" "unknown fish" etc. so i think it should stay in quotes